<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_SampleAnalysis">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_LastRecord">
            <property name="text">
             <string>上一记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NextRecord">
            <property name="text">
             <string>下一记录</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Print">
            <property name="text">
             <string>打印</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_SampleInfo">
            <property name="text">
             <string>样本信息输入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTabWidget" name="tabWidget">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>流程</string>
                </attribute>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>荧光数据</string>
                </attribute>
               </widget>
               <widget class="QWidget" name="tab_3">
                <attribute name="title">
                 <string>温度数据</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>样本信息</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_QC">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>270</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>QC</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_Set">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>200</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_ListReview">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>270</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>ListReview</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
